Index: Directory.Build.props
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Directory.Build.props b/Directory.Build.props
--- a/Directory.Build.props	(revision b1829b1a95da38bcd805f0bc039f8821d18ddb8d)
+++ b/Directory.Build.props	(date 1648269267546)
@@ -8,7 +8,7 @@
     <CredentialsTargetFramework>net6.0</CredentialsTargetFramework>
     <ServiceLayerTargetFramework>net6.0</ServiceLayerTargetFramework>
     <ResourceProviderTargetFramework>net6.0</ResourceProviderTargetFramework>
-    <ToolsServiceTargetRuntimes>win-x64;win-x86;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64;osx.10.11-x64;osx-x64;linux-x64</ToolsServiceTargetRuntimes>
+    <ToolsServiceTargetRuntimes></ToolsServiceTargetRuntimes>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
     <!-- This is required for IDE0005 to fail the build https://github.com/dotnet/roslyn/issues/41640 -->
